### DISCUSSION TITLE
What should be do with a Anti-Harrasment Policy

### DISCUSSION THREAD

**Discussion by albertico** [Code4PuertoRico/Code4PR#71] [Q&A]
As part of the Brigade Handbook we need to create an Anti-Harassment policy.

See the following [example from the Geek Feminism wiki](https://geekfeminism.wikia.org/wiki/Conference_anti-harassment/Policy).

**Comment by albertico** [comment]
Code for America has the Anti-Harassment policy merged into the Code of Conduct. Our current Code of Conduct includes that. Will take an overview look at it and determine whether that's enough for now.

Repo here: https://github.com/Code4PuertoRico/codeofconduct

**Comment by albertico** [comment]
The outstanding question is whether we want to keep it merged within the Code of Conduct or branch it into it's own document. cc: @froi

**Comment by froi** [comment]
I think that having it as it's own repo / document will help us iterate on it with community feedback easier/quicker.

Having our CoC link to a README in another repo might be enough for us to keep a document that is simple to read and edit when needed.

I see the CoC as a parent document with supporting documents like the anti-harrasment and Slack policies giving the details.
